lib/index.js

```diff
@@ -10,12 +10,15 @@ const outputChanges = require('./outputChanges')
 const outputResult = require('./outputResult')
 
 module.exports = async ({
+  config = false,
   dryRun = false,
   interactive = false,
   margin = 0,
   silent = false,
 } = {}) => {
-  const configPath = path.resolve(process.cwd(), 'jest.config.js')
+  const configPath = config
+    ? path.resolve(process.cwd(), config)
+    : path.resolve(process.cwd(), 'jest.config.js')
```

Comment:
If you apply the suggestions above, you don't need the conditional anymore:

```suggestion
  const configPath = path.resolve(process.cwd(), config)
```

Reply:
resolved in db585672e17064f8ff5c01d3451fa2ba48e131fd